eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/config/HttpRetryConfig.java

```diff
@@ -21,8 +21,8 @@
 
 @Data
 public class HttpRetryConfig {
-    // maximum number of retries, default 3, minimum 0
-    private int maxRetries = 3;
+    // maximum number of retries, default 2, minimum 0
+    private int maxRetries = 2;
```

Comment:
intended?

Reply:
What's the meaning? Previously, this field represented the maximum number of attempts, but considering that it existed in the retry configuration, it was changed to the maximum number of retries, but the corresponding default value was forgotten.

Reply:
OK